packages/cli/generation/ir-migrations/src/ir-versions/IrVersions.ts

```diff
@@ -43,8 +43,9 @@ export { FernIrV46 as V46 } from "@fern-fern/ir-v46-sdk";
 export { FernIrV47 as V47 } from "@fern-fern/ir-v47-sdk";
 export { FernIrV48 as V48 } from "@fern-fern/ir-v48-sdk";
 export { FernIrV49 as V49 } from "@fern-fern/ir-v49-sdk";
-export { FernIrV50 as V50 } from "@fern-fern/ir-v50-sdk";
 export * as V5 from "@fern-fern/ir-v5-model";
+export { FernIrV50 as V50 } from "@fern-fern/ir-v50-sdk";
+export { FernIrV51 as V51 } from "@fern-fern/ir-v51-sdk";
```

Comment:
@amckinney i don't see `irSerialization` updated -- is that something we want to add as well ?